src/macros.rst

```diff
@@ -151,7 +151,8 @@ a :t:`metavariable`.
 
 :dp:`fls_8o9mcV2KrKac`
 :t:`Fragment specifier` kinds impose the following
-:dt:`[fragment specifier restruction]s` on the :t:`[token]s` that follow them:
+:dt:`[fragment specifier restriction]s` on the :t:`[token]s` that follow them
+at the :t:`macro invocation` site:
```

Comment:
That's not correct, these restrictions are checked at the macro declaration site, https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=42e223f70885d65d3a3e8b63b980b730